### PR TITLE
example for feature #89

### PR DESCRIPTION
**Pull request by jogibear9988**
# Types of Changes

## Prerequisites

Please make sure you can check the following two boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project

## Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed

## Description

example for feature of minification of css #89

### CODE REVIEW

File: src/AngleSharp.Css/Constants/PropertyNames.cs [hunk 1]
```diff
@@ -10,1341 +10,1341 @@ public static class PropertyNames
         /// <summary>
         /// The align-content declaration.
         /// </summary>
-        public static readonly String AlignContent = "align-content";
+        public const String AlignContent = "align-content";
```
Comment (FlorianRappl) [inline src/AngleSharp.Css/Constants/PropertyNames.cs:13]: These are `public`. Can you keep them `static readonly`?

**Review by FlorianRappl — CHANGES_REQUESTED** [review]

**Comment by jogibear9988** [comment]
I changed them to "const" so I could use them in a switch. But this could be reverted. Maybe it also works with new patern matching switch without that change.

What do you think about the overall concept? Is this a way to do Simplification/Minification of CSS as part of AngleSharp.Css ?

**Comment by FlorianRappl** [comment]
> What do you think about the overall concept? Is this a way to do Simplification/Minification of CSS as part of AngleSharp.Css ?

Yeah we can leave it like that - personally I think for consistency having a dedicated formatter would make sense, too, but this also works.

> I changed them to "const" so I could use them in a switch. But this could be reverted. Maybe it also works with new patern matching switch without that change.

Understood - yes, I think it does work in the pattern matching style using references.

**Comment by FlorianRappl** [comment]
Will be part of 1.0.0

**Pull request closed without merging**